src/app/projects/projects.service.ts

```diff
@@ -35,7 +35,7 @@ export class ProjectsService {
     try {
       return {
         uuid: obj.uuid.uuid,
-        dcpUrl: obj.dcp_url,
+        dcpUrl: obj.wranglingState === 'Published in DCP' && `https://data.humancellatlas.org/explore/projects/${obj.uuid.uuid}`,
```

Comment:
do we want to create constants for the string literal?

Reply:
For what reason?